libs/community/langchain_community/chat_message_histories/file.py

```diff
@@ -20,21 +20,25 @@ def __init__(self, file_path: str) -> None:
         self.file_path = Path(file_path)
         if not self.file_path.exists():
             self.file_path.touch()
-            self.file_path.write_text(json.dumps([]))
+            with self.file_path.open("w", encoding="utf-8") as f:
```

Comment:
To add the options in a non breaking way, you'll need to add encoding and ensure ascii as part of the initializer.

If we set the default encoding to utf-8, we could break existing user message history (e.g., if their default encoding is a windows encoding)

Reply:
Thanks for the advice